tasks/makeSearchIndex.js

```diff
@@ -24,9 +24,14 @@ if (require.main !== module) {
       id: info.id,
       summary: info.summary,
       bom: info.bom.lines
-        .map(l => l.description)
+        .map(l => {
+          const description = l.description
+          const MPN = l.partNumbers.map(p => p.part)
+
+          return [description, MPN].filter(field => field !== '').toString()
+        })
         .filter(l => l !== '')
-        .toString()
+        .join('')
```

Comment:
No space between them!?

Reply:
There's a comma from inner `toString`

Reply:
Is comma the best seperator? How does the search actually handle the separation of words?

Reply:
I didn't modify the search mechanism, it uses `react-search-input` which doesn't use real indexing and searching mechanism it just matches the `term` with substrings of the `filters`. You said improving the search is a thing for the gitea version!

Reply:
Ok, I couldn't remember how it works actually. :joy: We should probably do something better, even before Gitea. But let's leave it for another issue/pull-request.